docs/configure/user-interface.md

```diff
@@ -0,0 +1,392 @@
+  background: theme.background.app,
+  width: 0,
+}));
+```
+
+Or with template literals:
+
+```js
+const Component = styled.div`
+  background: `${props => props.theme.background.app}`
+  width: 0;
+`;
+```
+
+### Storybook addons
+
+A key strength of Storybook is its extensibility. Storybook aims to be your tool of choice for developing components in the browser no matter what choice of UI framework your team has made. Another way that Storybook can fit your team’s development workflow is through addons that allow you to build better and more robust components.
+
+Addons have been a part of Storybook since the very beginning and are so integral to the way Storybook works that many of the features described in this documentation are implemented as addons! These addons are installed out of the box with [essentials](../essentials). 
+
+#### Addon features
+
+The most obvious thing addons affect in Storybook is the UI of Storybook itself. Within the UI the [toolbar]() and [addons panel]() are the two chief places addons will appear. 
+
+<div style="background-color:#F8FAFC">
+TODO: add image mentioned in the SB 6.0 needs to be further vetted
+</div>
+
+Addons can also hook into the rendering of your story in the preview pane via injecting their own [decorators](../writing-stories/decorators).
+
+Finally, addons can affect the build setup of Storybook by injecting their own webpack configuration to allow the use of other tools in Storybook. Addons that do only this are often referred to as [presets](../presets/introduction).
+
+### Essential, core and community addons
+
+There are many, many Storybook addons, but they can be roughly categorized into three areas:
+
+- Essential addons are core-team developed addons that are considered core to the out-of-the-box user experience of Storybook and thus ship by default with new Storybook installations.
+- Core addons are the addons that are developed by the core team with efforts made to keep them in sync with the development of Storybook itself, and to make sure they are written in idiomatic ways as templates for other addons. They can be found within the Storybook monorepo,
+- Community addons are addons written by the massive Storybook community. They can be found on our website, GitHub, and npm.
+
+### Sidebar & URLs
+
+Storybook’s sidebar lists all your stories grouped by component. When you have a lot of components you may wish to also group those components also. To do so, you can add the “/” separator to the “title” of your CSF file and Storybook will group the stories into groups based on common prefixes:
+
+<div style="background-color:#F8FAFC">
+TODO: add image mentioned in the SB 6.0 needs to be further vetted
+</div>
+
+Usually it is a good idea to use a nesting scheme that mirrors the filesystem path of the components. So if you have a file `components/modals/Alert.js` you might name the CSF file `components/modals/Alert.stories.js` and tile it “Components/Modals/Alert”.
```

Comment:
"Title it"

Reply:
this was another that slipped "2 pairs of eyes" (pardon the bad pun) :P